tests/Formatter/CurlFormatterTest.php

```diff
@@ -132,4 +132,31 @@ public function testProperBodyReading()
         $this->assertContains("-d 'foo=bar&hello=world'", $curl);
         $this->assertContains("-X PUT", $curl);
     }
+
+    /**
+     * @expectedException \Symfony\Component\Debug\Exception\FatalErrorException
+     *
+     * @dataProvider getHeadersAndBodyData
+     */
+    public function testextractBodyArgument($headers, $body)
```

Comment:
it should be CamelCase `testExtractB..` 
other than that I can't see any test performed inside this method